code/game/atoms_movable.dm

```diff
@@ -196,6 +196,12 @@
 	if(pulledby && moving_diagonally != FIRST_DIAG_STEP && get_dist(src, pulledby) > 1)		//separated from our puller and not in the middle of a diagonal move.
 		pulledby.stop_pulling()
 
+/atom/movable/proc/set_glide_size(target = 8)
+	glide_size = target
+
+	for(var/atom/movable/AM in buckled_mobs)
+		AM.set_glide_size(target)

```

Comment:
So since buckling doesn't update speed, the buckled mobs will be broken until they change speed ?

Reply:
yep, not anymore though.